features/lib/support/fake_wire_server.rb

```diff
@@ -67,13 +67,13 @@ def handle(data)
       end
     rescue => e
       response = [
-                    'fail',
-                    {
-                      :message => e.message,
-                      :backtrace => e.backtrace,
-                      :exception => e.class
-                    }
-                  ].to_json
+        'fail',
+        {
+          :message => e.message,
+          :backtrace => e.backtrace,
+          :exception => e.class
+        }
+      ].to_json
```

Comment:
I'm curious: Do we support Ruby < 1.9? Can we used `{ message: e.message }` colon syntax in Hashes? Are we already doing that?

Reply:
The lowest we support is 2.2 now. I can try changing it. :)

Reply:
@olleolleolle Hash syntax is handled by the Style/HashSyntax cop, but I'll change this one now. :)